Assets/Scripts/Player/Controller.cs

```diff
@@ -252,6 +252,12 @@ private void OnRollEnd()
 
         private void OnCollisionEnter2D(Collision2D other)
         {
+            
+            if (other.gameObject.CompareTag("Enemy"))
+            {
+                LoseHitPoints(1);
+            }
```

Comment:
If the player stops moving, and let's the Emu push them, they only take 1 point of damage.
You should change this so that the player takes some knockback, or program invincibility frames

Reply:
I have added 2 seconds of invincibility with a warning to the player. I can't implement blowback that doesn't look wonky with the current movement system.